src/core/src/bootstrap/EnvLayer.py

```diff
@@ -64,20 +64,25 @@ def get_package_manager(self):
         """ Detects package manager type """
         ret = None
 
-        # choose default - almost surely one will match.
-        for b in ('apt-get', 'yum', 'zypper'):
-            code, out = self.run_command_output('which ' + b, False, False)
+        if self.platform.linux_distribution()[0] == Constants.AZURE_LINUX:
+            code, out = self.run_command_output('which tdnf', False, False)
```

Comment:
Please get UT coverage similar to the rest (should be a trivial add-on to existing).

Reply:
Also, what is this code going to do if it's Azure Linux with a non-zero return code? 

It'll simply report this (not ideal for this) --
_Error: Package manager configuration not supported - None_

Reply:
This entire class is not covered by UTs. Should this be taken as a separate PR to improve code coverage?

Reply:
Addressed